holmes/plugins/toolsets/runbook/runbook_fetcher.py

```diff
@@ -19,26 +19,92 @@
 # runbooks from external sources as well.
 class RunbookFetcher(Tool):
     toolset: "RunbookToolset"
+    available_runbooks: List[str] = []
 
     def __init__(self, toolset: "RunbookToolset"):
+        # Load available runbooks from catalog
+        from holmes.plugins.runbooks import load_runbook_catalog
+
+        catalog = load_runbook_catalog()
+        available_runbooks = []
+        if catalog:
+            available_runbooks = [entry.link for entry in catalog.catalog]
+
+        # If additional search paths are configured (e.g., for testing), also scan those for .md files
+        # The config will be passed through the additional_search_paths parameter in the RunbookToolset __init__
+        if (
+            hasattr(toolset, "_additional_search_paths")
+            and toolset._additional_search_paths
+        ):
+            import os
+
+            for search_path in toolset._additional_search_paths:
+                if os.path.isdir(search_path):
+                    # Find all .md files in the search path
+                    for file in os.listdir(search_path):
+                        if file.endswith(".md"):
+                            # Add the filename (without path) to available runbooks if not already there
+                            if file not in available_runbooks:
+                                available_runbooks.append(file)
+
+        # Build description with available runbooks
+        runbook_list = (
+            ", ".join([f'"{rb}"' for rb in available_runbooks])
+            if available_runbooks
+            else '"networking/dns_troubleshooting_instructions.md", "upgrade/upgrade_troubleshooting_instructions.md"'
```

Comment:
can you explain why we add those runbooks by default?
again I would change this to pull from the parent dir so we dont have to hardcode names every time we add a runbook, maybe something like
search_dir_for_filetype(os.path.dirname(os.path.abspath(__file__)), suffix=".md")